frontend/common/src/components/skills/SkillResults/SkillResults.tsx

```diff
@@ -16,9 +16,14 @@ export const SkillBlock: React.FunctionComponent<{
   const { id, name, description } = skill;
   const [isOpen, setIsOpen] = useState(false);
 
+  const definition = description?.[locale];
```

Comment:
frontend/common/src/helpers/localize.ts has a helper for get localized name, FYI

Reply:
I purposefully did not use it here because we need `undefined` for later checks:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/612aceedae13911a457ddc509209045f52e55e15/frontend/common/src/components/skills/SkillResults/SkillResults.tsx#L57

https://github.com/GCTC-NTGC/gc-digital-talent/blob/612aceedae13911a457ddc509209045f52e55e15/frontend/common/src/components/skills/SkillResults/SkillResults.tsx#L101